nix/ci.nix

```diff
@@ -0,0 +1,7 @@
+{
+  default = import ../default.nix {};
+  # https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/haskell-packages.nix#L47
+  shell = (import ../shell.nix {});
+
+
```

Comment:
in here you can add more expressions and they'll get build by ci.
but you can also build it locally with `nix-build ci.nix`.